config/prod/prow/testgrid/testgrid.yaml

```diff
@@ -816,153 +816,6 @@ test_groups:
     alert_mail_to_addresses: "serverless-engprod-sea@google.com"
   alert_stale_results_hours: 170
   num_failures_to_alert: 1
-- name: ci-knative-serving-continuous-beta-prow-tests
```

Comment:
Are these deletion intentional?

Reply:
That file should not have any changes. An earlier version of the PR made unwanted changes, it was because the global "metaData" variable was getting polluted. I fixed that, and now there should be no changes to this file.

Reply:
This file still shows changed, did you forget to commit something?

Reply:
Looks like my changes are causing this somehow, probably due to more global variable pollution. I'm investigating.